v-next/hardhat-build-system/package.json

```diff
@@ -61,6 +61,7 @@
   "devDependencies": {
     "@eslint-community/eslint-plugin-eslint-comments": "^4.3.0",
     "@ignored/hardhat-vnext-node-test-reporter": "workspace:^3.0.0-next.0",
+    "@nomicfoundation/hardhat-test-utils": "workspace:^",
```

Comment:
Contrary to the rest of our packages, we do `workspace:^` here because this is a private package, which we don't intend to publish, so versioning is not an issue.